lib/manager/nuget/util.ts

```diff
@@ -51,7 +51,12 @@ export async function getConfiguredRegistries(
   }
 
   logger.debug({ nuGetConfigPath }, 'found NuGet.config');
-  const nuGetConfig = await readFileAsXmlDocument(nuGetConfigPath);
+  // Get relative path to nuget config
+  const relativeNuGetConfigPath = nuGetConfigPath.replace(
```

Comment:
Could you instead add a new fs function called `findUpLocal` which wraps `findUp` but which returns a relative path, and only if it's within the localDir?

Reply:
Sounds good.